### PR TITLE
fixed abusive 3D rendering; fixed rerender on dashboard ticker;

### PR DESCRIPTION
**Pull request by IAmMorrow**

### CODE REVIEW

File: src/renderer/screens/dashboard/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import React, { useCallback, useMemo } from "react";
 import Box from "~/renderer/components/Box";
 import { accountsSelector } from "~/renderer/reducers/accounts";
-import BalanceSummary from "~/renderer/components/BalanceSummary";
```
Comment (gre) [inline src/renderer/screens/dashboard/index.js:5]: can you explain the shift to a new component and why BalanceSummary is not replaced by it? (twice code to maintain?)

**Review by gre — APPROVED** [review]

File: src/renderer/screens/dashboard/index.js [hunk 2]
```diff
@@ -67,21 +66,6 @@ const DashboardPage = ({ saveSettings }: Props) => {
...
-        unit={counterValue.units[0]}
-        isAvailable={portfolio.balanceAvailable}
-        since={selectedTimeRange}
-        valueChange={portfolio.countervalueChange}
-        totalBalance={portfolio.balanceHistory[portfolio.balanceHistory.length - 1].value}
-        handleChangeSelectedTime={handleChangeSelectedTime}
-      />
-    ),
-    [t, counterValue.units, selectedTimeRange, handleChangeSelectedTime],
-  );
```
Comment (gre) [inline src/renderer/screens/dashboard/index.js:83]: well yeah that definitely sounded like a hacky idea

File: src/renderer/screens/dashboard/index.js (hunk 1 above)
Comment (IAmMorrow) [reply src/renderer/screens/dashboard/index.js:5]: Currently this component is duplicated there and there (account has its own for instance). I think it need to be redesigned. I did not take the time to include it in this PR.

File: src/renderer/screens/dashboard/index.js (hunk 2 above)
Comment (IAmMorrow) [reply src/renderer/screens/dashboard/index.js:83]: Yep that was a bit mind blowing

File: src/renderer/screens/dashboard/index.js (hunk 1 above)
Comment (gre) [reply src/renderer/screens/dashboard/index.js:5]: @IAmMorrow can you add a reminder in our spreadsheet?

**Pull request merged**